relay-server/src/actors/envelopes.rs

```diff
@@ -2715,8 +2715,24 @@ impl Handler<SendMetrics> for EnvelopeManager {
         let mut envelope = Envelope::from_request(None, RequestMeta::outbound(dsn));
         envelope.add_item(item);
 
+        let mut sizes: Vec<(usize, f64)> = Vec::new();
+        buckets.iter().for_each(|bucket| {
+            sizes.push((bucket.value.relative_size(), bucket.value.num_metrics()));
+        });
+
         let future = self
             .send_envelope(project_key, envelope, scoping, Instant::now())
+            .then(move |x| {
+                sizes.iter().for_each(|(rel_size, num_metrics)| {
+                    relay_statsd::metric!(
```

Comment:
Can't we log these in the aggregator flush to avoid exposing such internals from `relay-metrics`?